scriptmodules/ports/ikemen-go.sh

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env bash
+
+# This file is part of The RetroPie Project
+#
+# The RetroPie Project is the legal property of its developers, whose names are
+# too numerous to list here. Please refer to the COPYRIGHT.md file distributed with this source.
+#
+# See the LICENSE.md file at the top-level directory of this distribution and
+# at https://raw.githubusercontent.com/RetroPie/RetroPie-Setup/master/LICENSE.md
+#
+
+rp_module_id="ikemen-go"
+rp_module_desc="I.K.E.M.E.N GO - Clone of M.U.G.E.N to the Go programming language"
+rp_module_licence="MIT https://raw.githubusercontent.com/Windblade-GR01/Ikemen-GO/master/License.txt"
+rp_module_help="Copy characters, stages, screenpacks, etc. to $romdir/ports/ikemen-go\n\nConfig files can be found at $configdir/ports/ikemen-go/save"
+rp_module_section="exp"
+
+function depends_ikemen-go() {
+    getDepends golang-go libgl1-mesa-dev xinit xorg xorg-dev libopenal1 libopenal-dev libgtk-3-dev libasound2-dev freeglut3-dev
+}
+
+function sources_ikemen-go() {
+    gitPullOrClone "$md_build" https://github.com/Windblade-GR01/Ikemen-GO.git
+}
+
+function build_ikemen-go() {
+    go clean
+    go mod tidy
+    make TAGS='-tags al_cmpt' linux
+    # grabs default screenpack and content required for the game to run; note that the screenpack has a CC-BY-NC 3.0 license
+    make elecbyte
+    md_ret_require="$md_build/Ikemen_GO"
+}
+
+function install_ikemen-go() {
+    cp 'elecbyte/LICENCE.txt' 'ScreenpackLicense.txt'
+    mkdir save sound
```

Comment:
Is the save folder for writing? If so it is better to symlink it in the configure stage to `$md_conf_root/system` etc?

Reply:
The `save` folder is indeed written to by the engine; it's used for saving a few different things, like `config.json`, replay files, and a few other miscellaneous files in certain modes like `randomtest`.

The script now symlinks it to the `$configdir/ports/ikemen-go/save` folder, created with user-ownership via `mkUserDir` so that IKEMEN can write the files successfully.